src/utils/defaults.ts

```diff
@@ -363,7 +365,7 @@ export const defaultManageClientUsernameConfig:any = {
 
 export const defaultFeedConfig:any = {
   feedName: "stm-asyncapi_feed",
-  feedType: "asyncapi",
+  feedType: "asyncapi_feed",
```

Comment:
does this change mean we need to update existing feed files as well?

Reply:
No, it won't. Checked it..